lib/cli/user.js

```diff
@@ -0,0 +1,43 @@
+const logger = require('../helpers/logger')
+const MongoClient = require('../core/clients/mongodb-client')
+const UserCommands = require('../core/commands/user-commands')
+
+exports.command = 'users'
+
+exports.describe = 'Sets up users against any mongo instance'
+
+exports.builder = {
+}
+
+//  MongoError {
+//   code: 11000,
+//   codeName: 'DuplicateKey',
+//   ok: 0,
+//   message: 'User "david@admin" already exists',
+// }
+
+exports.handler = async (argv) => {
+  logger.debug('Setting up users')
+  logger.debug(argv)
+
+  const { uri } = argv.connection
+  const { users } = argv
+  const client = new MongoClient(uri)
+  await client.connect()
+  const userCommands = new UserCommands(client)
+
+  for (let i = 0; i < users.length; i++) {
+    const user = users[i]
+    logger.debug('creating user', user)
+    try {
```

Comment:
I think this should "get" the user before creating it instead:

- Get user
- If user exists: update user 
- if user does not exist: create user

Reply:
When I move this into user state I will do this too.